src/commands/coverage/__tests__/upload.test.ts

```diff
@@ -302,17 +307,6 @@ describe('execute', () => {
     expect(output[2]).toContain(`Will upload code coverage report file ${path}`)
   })
 
-  test('with git metadata without argument (default value is true)', async () => {
-    const {context, code} = await runCLI([
-      '--verbose',
-      process.cwd() + '/src/commands/coverage/__tests__/fixtures/single_file.xml',
-    ])
-    const output = context.stdout.toString().split(os.EOL)
-    expect(id).toHaveBeenCalled()
-    expect(code).toBe(0)
-    expect(output[4]).toContain('Syncing git metadata')
-  })
-
   test('without git metadata (with argument)', async () => {
```

Comment:
have we tested with folders that include `.xml`? Like 
`src/commands/coverage/__tests__/fixtures/reports.xml/my-report.xml` or even an empty folder like 
`src/commands/coverage/__tests__/fixtures/empty-folder.xml/`. I know I had issues with this in the past

Reply:
renamed `subfolder` to `subfolder.xml` to cover this case